.github/workflows/test-code.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           dotnet-version: |
             3.1.x
             5.0.x
-            6.0.x
+            6.0.303
```

Comment:
Api approvals broke with the .NET 6.0.400 SDK released today.  Not sure why; they run fine on my PC.  For some reason it would seem that the api approvals generate differently on Linux than Windows.

Reply:
https://github.com/PublicApiGenerator/PublicApiGenerator/issues/245